test/e2e/basic_test.go

```diff
@@ -593,3 +593,50 @@ func TestDeletePreviousRoundFlowsOnStartup(t *testing.T) {
 		t.Errorf("Flow was still present after timeout")
 	}
 }
+
+// TestGratuitousARP verifies that we receive 3 GARP packets after a Pod is up.
+// There might be ARP packets other than GARP sent if there is any unintentional
+// traffic. So we just check the number of ARP packets is greater than 3.
+func TestGratuitousARP(t *testing.T) {
+	data, err := setupTest(t)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer teardownTest(t, data)
+
+	podName := "busybox"
+	nodeName := masterNodeName()
```

Comment:
why the master?

Reply:
I wanted the test not limited by node number.
Now I see some tests are using `workerNodeName(1)` without checking node number, changed to `workerNodeName(1)` too.